paddle/fluid/operators/controlflow/conditional_block_infer_op.cc

```diff
@@ -51,7 +51,9 @@ class ConditionalBlockInferOp : public ConditionalOp {
 
     if (need_run) {
       auto *scope_var = scope.FindVar(Output("Scope"));
-      PADDLE_ENFORCE(scope_var != nullptr, "Must set scope");
+      PADDLE_ENFORCE_NE(scope_var, nullptr,
+                        platform::errors::InvalidArgument(
+                            "Scope must be set in ConditionalBlockInferOp"));
```

Comment:
May be better to use `PADDLE_ENFORCE_NOT_NULL`  and `platform::errors::PreconditionNotMet`.

Reply:
Done